v2/internal/testcommon/vcr/redact.go

```diff
@@ -128,7 +136,7 @@ var passwordMatcher = regexp.MustCompile("\"pass[^\"]*?pass\"")
 
 // hidePasswords hides anything that looks like a generated password
 func hidePasswords(s string) string {
-	return passwordMatcher.ReplaceAllLiteralString(s, "\"{PASSWORD}\"")
+	return passwordMatcher.ReplaceAllLiteralString(s, "\"{REDACTED}\"")
```

Comment:
I think we want this to stay as `PASSWORD` so that we can see what the value was in the recording.

Plus, doing this will break many of the existing recordings.